test/integration/hmrc_manual_updates_test.rb

```diff
@@ -39,7 +39,7 @@ class HmrcManualUpdatesTest < ActionDispatch::IntegrationTest
 
     manual_specific_breadcrumbs = page.all(".gem-c-breadcrumbs")[1]
     within manual_specific_breadcrumbs do
-      assert page.has_link?(I18n.t("manuals.breadcrumb_contents"), href: @content_item["base_path"])
+      assert page.has_text?(I18n.t("manuals.breadcrumb_contents"))
```

Comment:
We'll want to keep this behaviour ('Contents' link for HMRC and normal manuals updates pages to the parent manual). 

As part of this I separated out the breadcrumbs method as it was getting a bit chaotic in one place, but missed updating the [updates module](https://github.com/alphagov/government-frontend/blob/main/app/presenters/content_item/manual_updates.rb).

We could probably just add:

```
def breadcrumbs
  [
    {
      title: I18n.t("manuals.breadcrumb_contents"),
      url: base_path,
    },
  ]
end
```

Reply:
Thanks, updated and removed the related test changes.